tests/components/spotify/fixtures/playlist.json

```diff
@@ -153,13 +153,11 @@
         },
         "track": {
           "preview_url": "https://p.scdn.co/mp3-preview/d61fbb7016904624373008ea056d45e6df891071?cid=c7c59b798aab4892ac040a25f7dd1575",
-          "available_markets": [],
```

Comment:
They usually come with every object,  but that causes every fixture to have 184 country names per object extra, causing a few thousand lines of fixture. Since there was no significance, i just removed all of them to be consistent